packages/dev/docs/pages/react-spectrum/styling.mdx

```diff
@@ -186,6 +187,8 @@ The list of Spectrum color values is visualized below. Use the picker to see how
 light and dark color schemes.
 
 ```tsx import
+import ReactDOM from 'react-dom';
```

Comment:
oh I think I changed this to import `react-dom/client`: https://github.com/adobe/react-spectrum/blob/9790448803156f5f1412819164b7750bcf6b48fe/packages/dev/parcel-transformer-mdx-docs/MDXTransformer.js#L474

So the later line needs to change to `ReactDOM.createRoot(document.getElementById('color-viewer')).render(<ColorViewer />)`

Reply:
Both ways seemed to work, but I went ahead and updated it to this way.

Reply:
did you push? i still see it importing from react-dom, not the client subpath

Reply:
ya, `createRoot` is the React 18 way. `ReactDOM.render` throws a warning in the console.

Reply:
right, but i thought it was preferable to do
`import { createRoot } from "react-dom/client";`
?

wasn't aware that you could still get at it through just `react-dom`

Reply:
I can't get the second one to render with `createRoot` for some reason. Looking into it.

Reply:
Just updated, the culprit was a missing semicolon after the first render statement.